### PR TITLE
Progress bar

### PR DESCRIPTION
**Pull request by elesbom**

### CODE REVIEW

File: dbaas/logical/templatetags/capacity.py [hunk 1]
```diff
@@ -19,3 +24,207 @@ def render_capacity_html(database):
...
+
+    def _make_disk_part(self, **kw):
+        self.disk_parts.append(kw)
+
+    def _make_memory_part(self, **kw):
+        self.memory_parts.append(kw)
+
+    @cached_property
+    def total_disk_in_gb(self):
+        host_attr = self.instance.hostname.nfsaas_host_attributes.first()
```
Comment (otherpirate) [inline dbaas/logical/templatetags/capacity.py]: .filter(is_active=True)

File: dbaas/logical/templatetags/capacity.py [hunk 2]
```diff
@@ -19,3 +24,207 @@ def render_capacity_html(database):
...
+        total_disk_in_gb = (host_attr.nfsaas_size_kb or 0.0) * (1.0 / 1024.0 / 1024.0)
```
Comment (otherpirate) [inline dbaas/logical/templatetags/capacity.py]: Please use const GB_FACTOR from logical.models

File: dbaas/logical/templatetags/capacity.py [hunk 3]
```diff
@@ -19,3 +24,207 @@ def render_capacity_html(database):
...
+
+    @cached_property
+    def used_disk_in_gb(self):
+        used_disk_in_gb = self.databaseinfra.disk_used_size_in_gb
+        return self.normalize_number(used_disk_in_gb) if used_disk_in_gb is not None else None
+
+    @cached_property
+    def used_database_in_gb(self):
+        if self.is_in_memory and not self.is_persisted:
+            # TODO: Verificar se retorna 0 ou não mostra a barra
```
Comment (otherpirate) [inline dbaas/logical/templatetags/capacity.py]: Remove this comment

File: dbaas/logical/templatetags/capacity.py [hunk 4]
```diff
@@ -19,3 +24,207 @@ def render_capacity_html(database):
...
+        if self.used_disk_in_gb is None:
+            free_disk_in_gb = self.total_disk_in_gb - self.used_database_in_gb
+        else:
+            free_disk_in_gb = self.total_disk_in_gb - self.used_disk_in_gb
+
+        return self.normalize_number(free_disk_in_gb)
+
+    @cached_property
+    def database_percent(self):
+        database_percent = self.used_database_in_gb * 100 / self.total_disk_in_gb
```
Comment (otherpirate) [inline dbaas/logical/templatetags/capacity.py]: (self.used_database_in_gb * 100) / self.total_disk_in_gb

File: dbaas/logical/templatetags/capacity.py [hunk 5]
```diff
@@ -19,3 +24,207 @@ def render_capacity_html(database):
...
+
+        return self.normalize_number(database_percent)
+
+    @cached_property
+    def other_percent(self):
+        if self.used_disk_in_gb is None or self.is_in_memory:
+            other_percent = 0
+        else:
+            other_percent = self.used_other_in_gb * 100 / self.total_disk_in_gb
```
Comment (otherpirate) [inline dbaas/logical/templatetags/capacity.py]: (self.used_other_in_gb * 100) / self.total_disk_in_gb

**Comment by elesbom** [comment]
I do other modifications

**Pull request merged**